tests/integration/Utils.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace OpenStack\Integration;
+
+use GuzzleHttp\Client;
+use OpenStack\Identity\v2\Api;
+use OpenStack\Identity\v2\Service;
+use OpenStack\Common\Transport\HandlerStack;
+
+class Utils
+{
+    public static function getAuthOptsV3()
+    {
+        return [
+            'authUrl' => getenv('OS_AUTH_URL'),
+            'region'  => getenv('OS_REGION_NAME'),
+            'user'    => [
+                'id'       => getenv('OS_USER_ID'),
+                'password' => getenv('OS_PASSWORD'),
+            ],
+            'scope'   => [
+                'project' => [
+                    'id' => getenv('OS_PROJECT_ID'),
+                ]
+            ]
+        ];
+    }
+
+    public static function getAuthOptsV2()
+    {
+        $authUrl = \OpenStack\Common\Transport\Utils::normalizeUrl(getenv('OS_AUTH_URL'));
```

Comment:
We cannot, because the class name here is Utils too. I refactor those methods under OpenStack\Common\Transport so it will solve this normalizeUrl call.